azurelinuxagent/common/event.py

```diff
@@ -305,23 +306,23 @@ def __init__(self):
 
         # Parameters from OS
         osutil = get_osutil()
-        self._common_parameters.append(TelemetryEventParam("OSVersion", EventLogger._get_os_version()))
-        self._common_parameters.append(TelemetryEventParam("ExecutionMode", AGENT_EXECUTION_MODE))
-        self._common_parameters.append(TelemetryEventParam("RAM", int(EventLogger._get_ram(osutil))))
-        self._common_parameters.append(TelemetryEventParam("Processors", int(EventLogger._get_processors(osutil))))
+        self._common_parameters.append(TelemetryEventParam(CommonTelemetryEventSchema.OSVersion, EventLogger._get_os_version()))
+        self._common_parameters.append(TelemetryEventParam(CommonTelemetryEventSchema.ExecutionMode, AGENT_EXECUTION_MODE))
+        self._common_parameters.append(TelemetryEventParam(CommonTelemetryEventSchema.RAM, int(EventLogger._get_ram(osutil))))
+        self._common_parameters.append(TelemetryEventParam(CommonTelemetryEventSchema.Processors, int(EventLogger._get_processors(osutil))))
 
         # Parameters from goal state
-        self._common_parameters.append(TelemetryEventParam("VMName", "VMName_UNINITIALIZED"))
-        self._common_parameters.append(TelemetryEventParam("TenantName", "TenantName_UNINITIALIZED"))
-        self._common_parameters.append(TelemetryEventParam("RoleName", "RoleName_UNINITIALIZED"))
-        self._common_parameters.append(TelemetryEventParam("RoleInstanceName", "RoleInstanceName_UNINITIALIZED"))
+        # self._common_parameters.append(TelemetryEventParam("VMName", "VMName_UNINITIALIZED"))
```

Comment:
why commented out?

Reply:
Sorry forgot to remove, thanks for catching it!

Reply:
Thanks for the update; forgot to mention there was another instance:
```

$ find . -name '*.py' -exec grep -l '#.*"VMName"' {} \;
./azurelinuxagent/common/event.py
$ find . -name '*.py' -exec grep -n '#.*"VMName"' {} \;
357:            # parameters["VMName"].value = vminfo.vmName

```

Reply:
Done